src/Blog/View/posts.twig

```diff
@@ -12,14 +12,52 @@
 	<body class="top-wave">
 	{% endblock %}
 	<section class="container">
-		<h1>Blog</h1>
 		<ul>
-			{% for post in posts %}
-				<li>
-					<a href="/blog/{{ post.id }}">{{ post.title }}</a>
-				</li>
-			{% endfor %}
-		</ul>
+		{% for post in posts %}
+			<li class="u-margin--medium clearfix">
+				<a href="/blog/{{ post.id }}"><img src="" class="post-box__img" alt="alternative text"></img></a>
+				<div class="post-box__description">
+					<a href="/blog/{{ post.id }}" class="post-description__header clearfix">
+						<h2 class="post-description__title">{{ post.title }}</h2>
+						<span class="post description__date"></span>
+					</a>
+					<p class="post-description__preamble module line-clamp">It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).</p>			
+					<a href="/blog/{{ post.id }}" class="read-more-button clearfix">
+						<span>Zobacz więcej</span>
+						<img src="" class="move-forward-icon" alt="alternative text"></img>
+					</a>
+				</div>
+			</li>
+						<li class="u-margin--medium clearfix">
```

Comment:
open and close tag should be at the same position, why it is not like the li above ?